### ISSUE TITLE
Files for the case

### ISSUE THREAD

**Issue by AGSpec** [tjeffree/mininovaii-keyboard#2]
I seems the files for the case are corrupted and/or absent. Do you happen to still have  a copy?

**Comment by tjeffree** [comment]
Oof been a while. Think I just used the dxf files. Though it would have been useful if I'd recorded the heights for each layer, right?

**Comment by AGSpec** [comment]
Yes, if you could find or measure the height layer of each parts it would be nice. If thats not an option i'll try by trial and error. The only one thats pretty important is the switchplate.
